flixel/tweens/FlxTween.hx

```diff
@@ -621,7 +718,7 @@ typedef TweenOptions = {
 	?onUpdate:TweenCallback,
 	?onComplete:TweenCallback,
 	?startDelay:Null<Float>,
-	?loopDelay:Null<Float>,
```

Comment:
Interesting, didn't know Haxe allowed a trailing comma here for the last element. :)